integrations/anthropic/src/ucai_anthropic/utils.py

```diff
@@ -0,0 +1,146 @@
+from typing import Any, Dict, List, Optional
+
+from anthropic.types import Message, ToolUseBlock
+from pydantic import BaseModel, Field, ValidationError
+from ucai.core.client import BaseFunctionClient
+from ucai.core.utils.client_utils import validate_or_set_default_client
+from ucai.core.utils.function_processing_utils import construct_original_function_name
+
+
+class ConversationMessage(BaseModel):
+    role: str = Field(..., description="The role of the message sender, e.g., 'user' or 'assistant'.")
+    content: Any = Field(..., description="The content of the message, can be text, tool usage, or tool result.")
```

Comment:
Can we use Union[str, ...] to clarify on the type instead of Any?

Reply:
Updated with the supported types